capstone-ui/src/components/Navbar/Navbar.jsx

```diff
@@ -28,21 +31,16 @@ const Navbar = () => {
           </p>
         </div>
         <div className="navbar-login">
-          <button type="button"> Sign up </button>
-          <button type="button" onClick={this.requestProfile}>
+          <button type="button" onClick={""}>
```

Comment:
This is producing the console error about a function needed here instead of a string.